packages/graphiql/resources/renderExample.js

```diff
@@ -83,6 +83,7 @@ function getSchemaUrl() {
 // how you can customize GraphiQL by providing different values or
 // additional child elements.
 const root = ReactDOM.createRoot(document.getElementById('graphiql'));
+const graphqlVersion = GraphiQL.GraphQL.version;
```

Comment:
found how to easily get version of `graphql-js` without importing `package.json` or injecting code